sigs.yaml

```
@@ -3018,9 +3018,6 @@ sigs:
       slack: sig-storage-cosi
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/OWNERS
```

Ditto the other question I had: https://github.com/kubernetes/community/pull/8195/files#r1860849539

```suggestion
    - https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface/master/OWNERS
```